src/components/dashboard/main-content/course-enrollments/course-cards/BaseCourseCard.jsx

```diff
@@ -321,9 +321,11 @@ class BaseCourseCard extends Component {
   };
 
   renderOrganizationName = () => {
-    const { orgName } = this.props;
+    const { orgName, mode } = this.props;
+
+    const isExecutiveEducation2UCourse = EXECUTIVE_EDUCATION_COURSE_MODES.includes(mode);
     if (orgName) {
-      return <p className="mb-0">{orgName}</p>;
+      return <p className="mb-0">{orgName} {isExecutiveEducation2UCourse && <>&#x2022; Executive Education</>}</p>;
```

Comment:
It would be good if we can move `&#x2022;` to a constant.

Reply:
@muhammad-ammar `&#x2022;` is a valid HTML character, wouldn't it be better to keep it as is?